src/Mercure/Service/HubService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Mercure\Service;
+
+use Pimcore\Bundle\StudioBackendBundle\Mercure\Util\Constants\Mercure;
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\Mercure\HubInterface;
+
+/**
+ * @internal
+ */
+final readonly class HubService implements HubServiceInterface
+{
+    public function __construct(
+        private HubInterface $clientHub,
+        private int $cookieLifetime = 3600
```

Comment:
do we need this default value here, if the default value is set in the configuration?

Reply:
Yeah if it is not dependency injected it still has the same value, but yeah in general we could scrap it, but it also does not hurt imho